battery-calcuator/index.html

```diff
@@ -0,0 +1,508 @@
+
+            //rxcapacity
+            resultValues.rxcapacity = roundToTwoDecimals((rx_duration * transmissionInput.value * setupVars.RX_POWER) / (60 * 60));
+
+            //sleepcapacity
+            resultValues.sleepcapacity = roundToTwoDecimals((sleep_duration * transmissionInput.value * setupVars.SLEEP_POWER) / (60 * 60));
+
+            //capacities
+            let capacityCalc = ((tx_duration * transmissionInput.value * tx_power) + (rx_duration * transmissionInput.value * setupVars.RX_POWER) + (sleep_duration * setupVars.SLEEP_POWER)) / (60 * 60);
+
+            if(downloadFrequency.value == 1) {
+                resultValues.dailycapacity = roundToTwoDecimals(capacityCalc);
+                resultValues.weeklycapacity = roundToTwoDecimals(capacityCalc * 7);
+                resultValues.monthlycapacity = roundToTwoDecimals(capacityCalc * 30);
+            }
+            if(downloadFrequency.value == 7) {
+                resultValues.dailycapacity = 0;
+                resultValues.weeklycapacity = roundToTwoDecimals(capacityCalc);
+                resultValues.monthlycapacity = roundToTwoDecimals(capacityCalc * 4);
+            }
+            if(downloadFrequency.value == 30) {
+                resultValues.dailycapacity = 0;
+                resultValues.weeklycapacity = 0;
+                resultValues.monthlycapacity = roundToTwoDecimals(capacityCalc);
+            }
+
+            //noErrors
+            nextTick(() => {
+                let el = document.querySelector('.output-section');
+                if(el) {
+                    el.scrollIntoView({ behavior: "smooth" })
+                }
+            })
+        }
+
+        //helper functions
+        const datarate = (range, fc) => {
+            if(range < 0) return none;
+
+            if (fc == setupVars.MF) {
+                if(range < 11) return 5000;
+                if(range < 101) return 3000;
+                if(range < 1001) return 2000;
+                if(range < 2001) return 5000;
+                if(range >=2001) return none;
+            }else if (fc == setupVars.HF) {
+                if(range < 11) return 25000;
+                if(range < 101) return 20000;
+                if(range < 1001) return 5000;
+                if(range >= 2001) return none;
+            }
+        }
+
+        const txpower = (range, fc) =>  {
+            if(range < 0) return none;
+
+            if (fc == setupVars.MF || fc == setupVars.HF) {
+                if(range < 10) return dbToWatts(-20, 45);
+                if(range < 100) return dbToWatts(-10, 45);
+                return dbToWatts(0, 45);
+            }
+        }
+
+        const dbToWatts = (db, watts) => Math.pow(10, db / 10) * watts;
+
+        const roundToTwoDecimals = (num) => parseFloat(num).toFixed(2);
+
+        const updatedOutputText = (textToFormat) => {
+            if(!textToFormat) return;
+            let inputInSeconds = roundToTwoDecimals(textToFormat);
```

Comment:
Should we round to no decimals here for display? Decimal seconds aren't useful for display only take space.